src/Kernel/CompiledMethod.class.st

```diff
@@ -960,6 +960,12 @@ CompiledMethod >> readsField: varIndex [
 	^ super readsField: varIndex 
 ]
 
+{ #category : #evaluating }
+CompiledMethod >> receiver: aReceiver withArguments: anArray executeMethod: anObject [
+	<primitive: 188>
```

Comment:
This method looks quite strange if defined in this class. You can't reuse it anywhere without actual method instance.

Reply:
yes, it is a bit of a hack, like the mirror primitive methods on Context.